src/uct/api/v2/uct_v2.h

```diff
@@ -198,6 +209,47 @@ typedef struct uct_md_mem_dereg_params {
 } uct_md_mem_dereg_params_t;
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Operation parameters passed to @ref uct_iface_is_reachable_v2.
+ */
+typedef struct uct_iface_is_reachable_params {
+    /**
+     * Mask of valid fields in this structure, using bits from
+     * @ref uct_iface_is_reachable_field_mask_t. Fields not specified in this
+     * mask will be ignored. Provides ABI compatibility with respect to adding
+     * new fields.
+     */
+    uint64_t                     field_mask;
+
+    /**
+     * Device address to check reachability to.
+     * This field is not relevant if iface_attr.dev_addr_len == 0.
```

Comment:
what does it mean "not relevant" ? that it should not be passed or ignored?

Reply:
fixed